code/_onclick/hud/plane_master.dm

```diff
@@ -38,9 +46,8 @@
 	blend_mode = BLEND_OVERLAY
 
 /obj/screen/plane_master/game_world/backdrop(mob/mymob)
-	clear_filters()
```

Comment:
I can't figure out why filters are being rebuilt in this proc. I don't like this at all, because that would mean that we'd have to somehow keep track of every filter that should be applied to this plane master from this location, which just doesn't feel right. 

Maybe this should be removed altogether and handled elsewhere?